.eslintignore

```diff
@@ -1,4 +0,0 @@
-node_modules/
```

Comment:
Why no .eslintignore anymore?

Reply:
Because I updated the project to use the more up-to-date [standardJS style guide](standardjs.com), check [this](https://github.com/DaniGuardiola/node-vault/commit/e2cdd59919d1db5a2722f81d84793443b87d46b7) and [this commit](https://github.com/DaniGuardiola/node-vault/commit/e5f441a66c365e4dfc6da6755a45256a728ead52)